### DISCUSSION TITLE
Is there a way to use Steam Workshop Tags?

### DISCUSSION THREAD

**Discussion by TheOGArc** [X2CommunityCore/xcom2-launcher#286] [Help] [answered]
Is there any way to make it where mods have the tags that are included for them from the steam workshop? I see that there is a tags section in the launcher but they are all completely blank, and I feel that the included steam tags would make organizing mods a lot easier.

**Comment by Dragon-32** [comment, marked as answer]
Try:
Right-click on a mod > Use workshop tags

Note I'm using the beta but I don't think the functionality's changed.

**Reply by TheOGArc** [reply]
shoot man I never even realized that, thanks thisll help sorting my mods go a lot quicker
